db/migrate/20190630034751_create_host_facets_reported_data_facets.rb

```diff
@@ -0,0 +1,16 @@
+class CreateHostFacetsReportedDataFacets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :host_facets_reported_data_facets do |t|
+      t.integer :host_id
```

Comment:
`t.reference`?

Reply:
And you can even
```suggestion
      #index should be default, do not hurt to be explicit though
      t.references :host, foreign_key: true, index: { unique: true } 
```
and you can drop the manual index and foreign key. It looks much cleaner this way.